src/Storage/StorageBucket.php

```diff
@@ -54,10 +54,10 @@ public function __getHeaders(): array
 	 *
 	 * @throws Exception
 	 */
-	public function __construct($api_key, $reference_id)
+	public function __construct($api_key, $reference_id, $domain, $scheme, $path)
```

Comment:
This signature needs to change. It should default some values to be easier to work with.
```
public function __construct($api_key, $reference_id, $domain = 'supabase.co', $scheme = 'https', $path = '/storage/v1')
```